main.tf

```diff
@@ -82,4 +83,16 @@ resource "aws_backup_vault_notifications" "default" {
   backup_vault_name   = aws_backup_vault.backup_vault.name
   sns_topic_arn       = var.vault_notification_sns_topic_arn
   backup_vault_events = var.backup_vault_events
+}
+
+# AWS Backup selection - selection resources
+resource "aws_backup_selection" "backup_selection_resources" {
+  count = var.account_type == local.account_type.workload ? 1 : 0
```

Comment:
Should we add the resource selection by ARN to the same backup selection block that already works with tags? https://github.com/DNXLabs/terraform-aws-backup/blob/master/main.tf#L62